code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1264,14 +1264,16 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/spec_fully_heal(mob/living/carbon/human/H)
 	return
 
+
 /datum/species/proc/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
-	if(!((target.health < 0 || HAS_TRAIT(target, TRAIT_FAKEDEATH)) && !(target.mobility_flags & MOBILITY_STAND)))
+	if(target.body_position == STANDING_UP || (target.health >= 0 && !HAS_TRAIT(target, TRAIT_FAKEDEATH)))
 		target.help_shake_act(user)
 		if(target != user)
 			log_combat(user, target, "shaken")
-		return 1
-	else
-		user.do_cpr(target)
+		return TRUE
+
+	user.do_cpr(target)
```

Comment:
This check is a little suspect, making a note for future use.

Reply:
The previous check was a bit convoluted with double negatives.
It went something like "if it's not true that the person is not-standing and it's not true that it has lower health than zero or has fake death, then shake the person, else one of those things is true and you can do cpr"
The new check should follow the same logic, removing the double negative: "if the person is standing up or has positive health while not having fake death, shake them, else try cpr"